workflows/microbiome/nanopore-allele-based-pathogen-identification/README.md

```diff
@@ -0,0 +1,14 @@
+# Nanopore _Allele-based Pathogen Identification_
+
+This workflow identifies pathogens using an allelic approach, detecting Single Nucleotide Polymorphisms (SNPs) to track emerging variants, i.e. markers showing evolutionary histories of homogeneous strains. This process includes SNP calling, aimed at identifying novel pathogen strains and elucidating discrepancies compared to reference sequences, thereby facilitating the tracking of emerging variants. Within this workflow, both variants and SNPs are discerned, serving as crucial elements for subsequent pathogen identification and variant tracking purposes.
+
+To perform the mapping step before variant identification, we used the Minimap2 tool, specifically designed for Nanopore reads. If you're working with Illumina data, you can substitute Minimap2 with Bowtie2.
```

Comment:
Minimap2 is not specifically design for nanopore data, it's mostly a tool for long reads that have a different error profile from short read data.

Reply:
Thank you so much for this comment, I missed that